### DISCUSSION TITLE
the indent problem

### DISCUSSION THREAD

**Discussion by weizhenhuan** [community/community#10789] [Copilot Conversations]
![image](https://user-images.githubusercontent.com/55399998/151727737-548e9043-8fcb-4444-a330-17ee74044f52.png)
![image](https://user-images.githubusercontent.com/55399998/151728302-4abc2c96-05f9-4f0a-9034-5e45916af802.png)

As you see, in Vscode, I set my tabSize 2, but the autocompleted code's indent is 4, I have to ajust the indent every time.

**Comment by davecheney** [comment]
Is copilot suggesting four spaces, not a single tab?

**Reply by weizhenhuan** [reply]
yes! Can you have a solution?

**Reply by KTibow** [reply]
If your code has no context, Copilot doesn't know how to write code your way.
Remember, Copilot is being fed code, not data (example: if your file is a Python file, it adds the shebang for it when getting suggestions). Right now, `editor.tabSize` doesn't change Copilot's behavior AFAIK.
